Proofs/Sha512_block_armv8.lean

```diff
@@ -4,6 +4,7 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Author(s): Shilpi Goel
 -/
 import Arm.Exec
+import Arm.State
```

Comment:
I'm curious why this import was needed. `Arm.Exec` should already import `Arm.State`.

Reply:
It was introduced while I was trying to resolve some error. Removed now